source/_integrations/huum.markdown

```diff
@@ -0,0 +1,26 @@
+---
+title: Huum
+description: Instructions on how to integrate a Huum saunas into Home Assistant.
+ha_category:
+  - Climate
+ha_release: 2022.6
```

Comment:
Needs to be updated when the release to include is known.

Reply:
fixed